.travis.yml

```diff
@@ -6,24 +6,25 @@ git:
 
 os:
   - linux
-  - osx
+  # - osx
 dist: bionic
 osx_image: xcode11
 
 env:
   global:  # default values
     - PYTHON_VERSION=3.7
   matrix:
-    - TASK=regular PYTHON_VERSION=3.6
-    - TASK=sdist PYTHON_VERSION=2.7
-    - TASK=bdist
-    - TASK=if-else
-    - TASK=lint
-    - TASK=check-docs
-    - TASK=mpi METHOD=source
-    - TASK=mpi METHOD=pip
-    - TASK=gpu METHOD=source PYTHON_VERSION=3.5
-    - TASK=gpu METHOD=pip PYTHON_VERSION=3.6
+    # - TASK=regular PYTHON_VERSION=3.6
+    # - TASK=sdist PYTHON_VERSION=2.7
+    # - TASK=bdist
+    # - TASK=if-else
+    # - TASK=lint
+    # - TASK=check-docs
+    # - TASK=mpi METHOD=source
+    # - TASK=mpi METHOD=pip
+    # - TASK=gpu METHOD=source PYTHON_VERSION=3.5
+    # - TASK=gpu METHOD=pip PYTHON_VERSION=3.6
```

Comment:
commenting out stuff to speed up the feedback cycle and minimize number of tasks running on Travis (since other PRs compete for those resources). Will revert once I feel this is in a working state